### PR TITLE
Merge the two things cause we updated stuff

### PR DESCRIPTION
**Pull request by DwarfLord42**
Title says it all pretty much.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -171,27 +176,43 @@ client.on(`message`, (message) => {
...
+    if(logBlackList[i]===message.channel.name){
+        return;
+    }
+}
   attachments = message.attachments.array().length!=0 ? "Yes" : "No";
+  
   util.logToServer(`The following message was deleted:
     Id: ${message.id}
-    Author: ${message.author}
+    Author: ${message.author.username}
```
Comment (Mercylas) [inline app.js]: We need the actual author link - not the username. Especially important with people changing their names.

File: app.js [hunk 2]
```diff
@@ -171,27 +176,43 @@ client.on(`message`, (message) => {
...
     Content: "${message}"
-    Attachments: ${attachments}`);
+    Attachments: ${attachments}
+    Channel: ${message.channel}`
```
Comment (Mercylas) [inline app.js]: Can we put channel before attachments?

File: app.js (hunk 1 above)
Comment (swim224) [reply app.js]: Done.

File: app.js (hunk 2 above)
Comment (swim224) [reply app.js]: Done.

**Pull request merged**